app/bundles/LeadBundle/Model/FieldModel.php

```diff
@@ -1021,32 +1019,6 @@ public function getGroupFields($group, $filters = ['isPublished' => true]): arra
         return $leadFields;
     }
 
-    /**
-     * Retrieves a list of published fields that are unique identifers.
-     *
-     * @deprecated to be removed in 3.0
-     *
-     * @return array<mixed>
-     */
-    public function getUniqueIdentiferFields($filters = []): array
-    {
-        return $this->getUniqueIdentifierFields($filters);
-    }
-
-    /**
-     * Retrieves a list of published fields that are unique identifers.
-     *
-     * @deprecated Use FieldsWithUniqueIdentifier::getFieldsWithUniqueIdentifier method instead
-     *
-     * @param array<mixed> $filters
-     *
-     * @return array<mixed>
-     */
-    public function getUniqueIdentifierFields(array $filters = []): array
```

Comment:
It's amazing to remove the usage of these deprecated methods. But they can be removed completely only in a major version. So this PR either can wait for M6 or if you revert these methods in this PR it can be merged into M5 and removed completely in M6 afterwards. What would you prefer?

Reply:
@escopecz well, it's 6.x time.